cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -201,6 +203,25 @@ impl RawManifest {
 
         manifest.insert(MANIFEST_PROFILE_KEY, Item::Table(profile_table));
 
+        if use_catalog {
```

Comment:
is the catalog feature relevant here?

Reply:
Yeah, there's details in a17319e9d7059b45bf7958dc1866ff307dc07761